graspologic/layouts/auto.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-from typing import Any, Dict, List, Tuple
+from typing import Any, Dict, List, Tuple, Optional
```

Comment:
Minor nit: can you keep these alphabetical?

Reply:
edit: just this line I mean, so `Any, Dict, List, Optional, Tuple`

Reply:
Ouch. I sorted all the imports.
Is it ok or shall i revert?

Reply:
Nope, that looks fine. There's some sort of like tribal knowledge around what gets clustered together and sorted in what order but it isn't written down and until it is, what you have is imo perfectly fine.

Maybe I'll start a discussion on this so we can put it in our contribution guide to help people out in the future!

Reply:
#630 is the discussion thread :)